analyzers/tests/SonarAnalyzer.UnitTest/TestCases/EmptyMethod.CSharp11.cs

```diff
@@ -26,3 +26,19 @@ public unsafe partial class Externals
     [System.Runtime.InteropServices.DllImportAttribute("ole32.dll", EntryPoint = "F", ExactSpelling = true)]
     private static extern partial void F(); // Compliant
 }
+
+// https://github.com/SonarSource/sonar-dotnet/issues/7629
+public class Repro_7629
+{
+    interface Interface_7629
+    {
+        void MyMethod();
+        static abstract void MyStaticAbstractMethod();
+    }
+
+    class MyClass_7629 : Interface_7629
```

Comment:
Add more test cases:
* explicit interface implementation
* A method that implements one method from two interfaces: interface I1 -> void Save();  interface I2-> void Safe(); class C: I1, I2 -> void Save() { }
* An interface implementation through some inheritance from metadata (Consider https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.comparer-1.compare as a starting point)